gulpfile.js

```diff
@@ -2,12 +2,4 @@ const gulp = require('gulp');
 const { build } = require('@justeat/gulp-build-fozzie');
 
 // Grab the Fozzie build tasks
-build(gulp, {
-    js: {
```

Comment:
Would it make sense to have this instead of removing it? Reason being is that for my CDN task I need the JS file to be a named file i.e `fozzie.js` or `fozzie.min.js` as apposed to just `scripts.js` etc. Teamcity has a build step that outputs the css and js files to S3, currently I have another repo outputting a scripted JS file (with the same name) so to avoid confusion I was going to remove it (after the discussion in hipchat earlier today) but as your add JS now it would be nice to add the below so it does it automatically ☺️ 

```javascript
build(gulp, {
    js: {
        files: {
            main: {
                srcPath: 'index.js',
                distFile: 'fozzie.js'
            }
        }
    }
});
```